<launch>

    <!-- Run our interface with the 3DPrinter -->
    <node pkg="octo_ros" type="octoROS.py" name="printerWatcher" output="screen" cwd="node" />

</launch>
